core/modules/qproc/testQueryAna.h

```diff
@@ -108,6 +104,8 @@ std::string buildFirstParallelQuery(QuerySession& qs, bool withSubChunks=true) {
     return first.queries[0];
 }
 
+} // anonymous namespace
```

Comment:
I'm not sure about utility of the anonymous namespaces in header files, if it needs to be "hidden" then I think our standard practice is to put it inside `detail` namespace.